crates/solidity/testing/sourcify/src/command.rs

```diff
@@ -51,15 +48,24 @@ pub struct ShowCombinedResultsCommand {
     pub results_file: PathBuf,
 }
 
+#[allow(clippy::struct_excessive_bools)]
 #[derive(Debug, Parser)]
 pub struct TestOptions {
     /// Run bindings tests.
     #[arg(long, default_value_t = false)]
     pub check_bindings: bool,
 
```

Comment:
should the new options be also exposed in `sourcify_all_chains.yml` and `sourcify_single_chain.yml`?